va/va.h

```diff
@@ -777,6 +777,18 @@ typedef enum
      * implementation, multiple frames encode/decode can improve HW concurrency
      */
     VAConfigAttribMultipleFrame         = 40,
+    /** \brief indicates the number of scalable pipes minus 1 that the underline encoder supports.
+     * Application may use this number to decide the number of tile columns if scalable pipes are used.
+     */
+    VAConfigAttribNumScalablePipes      = 41,
```

Comment:
it does not need to put something like "Minus1" in the parameter name?

Reply:
Done